plugins/org.python.pydev.jython/src/org/python/pydev/jython/JythonModules.java

```diff
@@ -52,13 +52,16 @@ private static PyObject getPepJythonModule(IPythonInterpreter interpreter, Strin
     public static String makeISort(String fileContents, File f, Set<String> knownThirdParty) {
         IPythonInterpreter iPythonInterpreter = iSortThreadLocalInterpreter.get();
         IPythonInterpreter interpreter;
-        String outputLine = "output = getattr(isort.SortImports(file_contents=fileContents, settings_path=settingsPath, known_third_party=knowhThirdParty), 'output', None)\n";
+        String outputLine = "output = getattr(isort.SortImports(file_contents=fileContents, settings_path=settingsPath, known_third_party=knownThirdParty), 'output', None)\n";
         if (iPythonInterpreter == null) {
             // The first call may be slow because doing the imports is slow, but subsequent calls should be
             // fast as we'll be reusing the same interpreter.
             String s = ""
                     + "import sys\n"
+                    + "import os\n"
                     + "add_to_pythonpath = '%s'\n"
+                    + "os.chdir(add_to_pythonpath)\n"
+                    + "add_to_pythonpath = os.getcwd()\n"
```

Comment:
I don't think this line is needed (i.e.: `add_to_pythonpath = os.getcwd()`).

If it's needed, can you elaborate a bit on it?

Reply:
Following https://www.tutorialspoint.com/python/os_chdir.htm use example, os.chdir(path) goes to the respectively path directory and os.getcwd() return it's path

i.e.: os.chdir(add_to_pythonpath) by itself does not change add_to_pythonpath

Reply:
Correction:
I did some tests and perhaps os.getcwd() is kinda redundant, so I remove it from the fix